ui/src/localStorage.js

```diff
@@ -1,29 +1,21 @@
-import _ from 'lodash';
-
 export const loadLocalStorage = () => {
   try {
     const serializedState = localStorage.getItem('state');
-    const defaultTimeRange = {upper: null, lower: 'now() - 15m'};
-
-    if (serializedState === null) {
-      return {timeRange: defaultTimeRange};
-    }
-
     const parsedState = JSON.parse(serializedState) || {};
-    const timeRange = _.isEmpty(parsedState.timeRange) ? defaultTimeRange : parsedState.timeRange;
 
-    return {...parsedState, timeRange};
+    return {...parsedState};
```

Comment:
This doesn't need to be unrolled, right? It's going to be a new immutable object either way, right?